generator/uscore/uscore_generator.rb

```diff
@@ -577,6 +577,10 @@ def create_multiple_or_test(sequence)
             reply = get_resource_by_params(versioned_resource_class('#{sequence[:resource]}'), search_params)
             validate_search_reply(versioned_resource_class('#{sequence[:resource]}'), reply, search_params)
             assert_response_ok(reply)
+            resources_returned = fetch_all_bundled_resources(reply.resource)
+            assert #{param_value_name(param)}.split(',').all? do |val|
```

Comment:
This assert needs a failure message. In order to have a helpful error message you'll probably need to track which values have been found. Something along the lines of:
```
missing_values = all_the_values.reject { |value| is_this_value_present_somewhere?(value) }
assert missing_values.empty?, 'helpful error message'
```

Reply:
fixed